### PR TITLE
Fix Dashboard UI in light mode

### PR DESCRIPTION
**Pull request by guanbinrui**
PC:

+ Fix the wrong background color of the drawer in light mode.
+ Fix the wrong background color of the popup page in light mode.
+ Dismiss the "decoding image failed" error message [RC](https://dimension.chat/group/maskbook?msg=k9HuqXmnWxeWbsGnx).

Mobile:

+ Adjust margin of the icons in the header bar.

### CODE REVIEW

File: src/extension/popup-page/index.tsx [hunk 1]
```diff
@@ -99,13 +99,12 @@ function Popup() {
...
+            ? MaskbookDarkTheme
+            : MaskbookLightTheme
     return (
-        <ThemeProvider
-            theme={
-                (preferDarkScheme && appearance === Appearance.default) || appearance === Appearance.dark
-                    ? MaskbookDarkTheme
-                    : MaskbookLightTheme
-            }>
+        <ThemeProvider theme={Object.create(theme)}>
```
Comment (Jack-Works) [inline src/extension/popup-page/index.tsx:107]: Reason?

File: src/extension/options-page/index.tsx [hunk 2]
```diff
@@ -135,65 +134,64 @@ function DashboardUI() {
...
+            : MaskbookLightTheme
     return (
         <I18nextProvider i18n={i18nNextInstance}>
-            <ThemeProvider
-                theme={
-                    (preferDarkScheme && appearance === Appearance.default) || appearance === Appearance.dark
-                        ? MaskbookDarkTheme
-                        : MaskbookLightTheme
-                }>
+            <ThemeProvider theme={Object.create(theme)}>
```
Comment (Jack-Works) [inline src/extension/options-page/index.tsx:187]: Why use Object.create?

Comment (guanbinrui) [reply src/extension/options-page/index.tsx:187]: This one isn't necessary. Just follow the change in the popup page. I will undo it.

File: src/extension/options-page/DashboardComponents/Carousel.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
+import { useInterval } from 'react-use'
+import { Fade } from '@material-ui/core'
+
+export interface CarouselProps {
+    items: React.ReactElement[]
+    delay?: number
+}
+
+export function Carousel({ items, delay = 1e4 }: CarouselProps) {
```
Comment (Jack-Works) [inline src/extension/options-page/DashboardComponents/Carousel.tsx:10]: Maybe `FadeInList`?

File: src/extension/popup-page/index.tsx (hunk 1 above)
Comment (guanbinrui) [reply src/extension/popup-page/index.tsx:107]: A piece of a snippet from MUI codebase:

```ts
  const theme = React.useMemo(() => {
    const output = outerTheme === null ? localTheme : mergeOuterLocalTheme(outerTheme, localTheme);

    if (output != null) {
      output[nested] = outerTheme !== null;
    }

    return output;
  }, [localTheme, outerTheme]);
```

MUI always merge theme into the previous one. For some reason, this memo didn't update. Use `Object.create()` to force it update.

File: src/extension/options-page/index.tsx (hunk 2 above)
Comment (Jack-Works) [reply src/extension/options-page/index.tsx:187]: It leads to a new theme every-render

File: src/extension/popup-page/index.tsx (hunk 1 above)
Comment (Jack-Works) [reply src/extension/popup-page/index.tsx:107]: It leads to a new theme object every-render

File: src/extension/options-page/index.tsx (hunk 2 above)
Comment (guanbinrui) [reply src/extension/options-page/index.tsx:187]: That's true. I'm removing it.

File: src/extension/options-page/DashboardComponents/Carousel.tsx (hunk 3 above)
Comment (guanbinrui) [reply src/extension/options-page/DashboardComponents/Carousel.tsx:10]: It's not about the list. It's fade items in the list.

File: src/extension/popup-page/index.tsx (hunk 1 above)
Comment (guanbinrui) [reply src/extension/popup-page/index.tsx:107]: It's intentional here.

Comment (Jack-Works) [reply src/extension/popup-page/index.tsx:107]: It's not mutating the previous theme https://github.com/mui-org/material-ui/blob/a3872cbb9cc95e91ab93875658d34fed1b878174/packages/material-ui-styles/src/ThemeProvider/ThemeProvider.js#L27 so I still not get why we need this hack here.

Comment (guanbinrui) [reply src/extension/popup-page/index.tsx:107]: A better workaround:
```
const theme = useMemo(
    () =>
        Object.create(
            (preferDarkScheme && appearance === Appearance.default) || appearance === Appearance.dark
                ? MaskbookDarkTheme
                : MaskbookLightTheme,
        ),
    [preferDarkScheme, appearance],
)
```

Comment (Jack-Works) [reply src/extension/popup-page/index.tsx]: Actually the 1st arg in `Object.create` means the _prototype_ of the new object. Why you need it on the prototype?

Comment (guanbinrui) [reply src/extension/popup-page/index.tsx]: It's OK with other methods to create a new object. Like `{...theme}`

File: src/extension/popup-page/index.tsx [hunk 4]
```diff
@@ -54,63 +68,59 @@ function Popup() {
...
-function PopupWithProvider() {
+function Popup() {
     const preferDarkScheme = useMediaQuery('(prefers-color-scheme: dark)')
     const appearance = useValueRef(appearanceSettings)
-    return (
-        <ThemeProvider
-            theme={
+    const theme = useMemo(
+        () =>
+            Object.create(
```
Comment (Jack-Works) [inline src/extension/popup-page/index.tsx:109]: Please use `Object.assign({}, ...)` or `{...}`

**Pull request merged**